quality-assurance/browser-automation.md

```diff
@@ -2,80 +2,93 @@
+Playwright enables fast, reliable and capable automation across all modern browsers. It is a Node.js library (also available in Python, Java and .NET) to automate Chromium, Firefox and WebKit with a single API. 
 
-One of the big disadvantages over Cypress is that it does not have a fully supported Test Runner ([playwright-test](https://github.com/microsoft/playwright-test) is currently in its first preview release), therefore screenshots, videos, DOM and network traffic are not recorded out of the box, making debugging tests less intuitive.
+Upon install, Playwright downloads a version of Chromium/Chrome and uses the Chrome DevTools Protocol to orchestrate the browser instance. For Firefox and WebKit engines, Playwright downloads the actual browser but extends their debugging protocol capabilities to provide a unified API and features. There is no modification of the actual browsers, so that it is expected to work exactly the same in the testing and the real user's browser. Go [here for more information on the browser binaries](https://playwright.dev/docs/browsers).
 
 The core advantage of using Playwright over Cypress is that it is an out-of-process automation driver that is not limited by the scope of in-page JavaScript execution and can automate scenarios with multiple pages and popups.
 
-Playwright's documentation and community is not as comprehensive as Cypress or Selenium, yet, it is constantly being added to as the tool is developed and the community is growing.
+Playwrights [documentation is excellent](https://playwright.dev/), tool is being continually developed and the community is growing.
 
-## Selenium
+### Selenium
 
 Selenium is a tool for web browser automation that uses WebDrivers to remotely control browser instances and emulate a user’s interaction with the browser. This allows tests to be run across many browsers and versions using the associated WebDriver.
 
-One of the reasons that Selenium is popular is because of its available language bindings, these include Java, C#, JavaScript and Python.
+Selenium is popular because of its long history, large community and available language bindings - these include Java, C#, JavaScript and Python.
 
-Selenium is a very good tool to automate true E2E and user interactions, but it takes time and effort to get the framework right, and then there is the overhead of maintaining it along with the WebDrivers and browser versions (though this overhead can be reduced through the use of tools such as [automated driver synchronisation](https://github.com/UKHO/ChromeDriverBinarySync-PSModule).
+Selenium is a very good tool to automate true E2E and user interactions, but it takes time and effort to get the framework right. It suffers  and then there is the overhead of maintaining it along with the WebDrivers and browser versions (though this overhead can be reduced through the use of tools such as [automated driver synchronisation](https://github.com/UKHO/ChromeDriverBinarySync-PSModule).
```

Comment:
"It suffers and then there is the overhead of maintaining"?

Reply:
Hahaha, I have no idea how that happened!